src/main/resources/application.yml

```diff
@@ -31,6 +33,8 @@ spring:
       hibernate:
         dialect: org.hibernate.dialect.PostgreSQLDialect
         format_sql: true
+        jdbc:
+          batch_size: 50
```

Comment:
I don't think this will actually do anything, since (afaik) we don't actually use hibernate in mod-fqm-manager. But I figured if we have some hibernate properties at all, we should also have this one :P